### ISSUE TITLE
Imps no longer spawn bats

### ISSUE THREAD

**Issue by geoffb** [lostdecade/onslaught_arena#205]
Imps don't spawn bats anymore. Probably related to the auto spawned object destruction fix. (To address projectiles killing the player after boss death)

The fix would be to create the bats and objects not owned by the imp (aka don't use spawnObject)

**Comment by geoffb** [comment]
Imps spawn bats again. Closed by f11eb18cf3192fe9f7beea314c68302020c71881

**Comment by geoffb** [comment]
Also, for future reference you can now avoid ownership of spawned objects by passing `false` as the 4th param to horde.Engine.spawnObject():

```
engine.spawnObject(parent, "monster_type", direction, takeOwnership);
```
